docs/Antora/modules/howto_guides/pages/howto_configure_instance.adoc

```diff
@@ -0,0 +1,291 @@
+= Instance configuration
+
+The Instance is the top-level openDAQ(TM) object, serving as a container for the openDAQ context and the base module manager.
+
+**Related articles:**
+
+  * xref:background_info:opendaq_architecture.adoc#instance[Architecture - Instance]
+
+== Creating the default openDAQ(TM) Instance
+Upon creation, it generates a Client device, which is a default device implementation capable of loading any function blocks present in the module manager's search path. If the native openDAQ client module is loaded, the Client device can connect to any TMS-enabled device using the `addDevice` function. The Client is set as the root device when the instance is created.
```

Comment:
I'd modify this a bit. The client device is capable of using the module manager to add function blocks, and connect to any device supported by the loaded modules. This is not OPC UA specific, but just depends on the modules loaded. Eg. you can connect to any streaming-only device that will not use OPC UA. 

Notably, "TMS" is a deprecated keyword, the OPC UA standard is now named the "openDAQ OPC UA" standard.